code/lib/csf-tools/src/enrichCsf.ts

```diff
@@ -25,3 +47,20 @@ export const extractSource = (node: t.Node) => {
   const { code } = generate.default(src, {});
   return code;
 };
+
+export const extractDescription = (node?: t.Node) => {
+  if (node?.leadingComments) {
+    const comments = node.leadingComments
+      .map((comment) => {
+        if (comment.type === 'CommentLine' || !comment.value.startsWith('*')) return null;
+        return comment.value
+          .split('\n')
+          .map((line) => line.replace(/^(\s+)?(\*+)?(\s+)?/, ''))
+          .join('\n')
+          .trim();
+      })
+      .filter(Boolean);
+    return comments.join('\n');
+  }
+  return '';
```

Comment:
nit: an early return will probably make this more readable, so you have to mentally parse less conditional indents. Like

```ts
if(!node?.leadingComments) {
  return '';
}
const comments = node.leadingComments;
...
```

Reply:
Good call! Fixed!